apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/TestDatabaseFixture.cs

```diff
@@ -12,57 +12,66 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#region
```

Comment:
let me alter my resharper settings and rerun code cleanup on my other PR.
Speaking of which, I'm investigating running resharper lint checks as jetbrains has a CLI version that can run as part of a CI system.  WDYT?